clusters/build-clusters/02_cluster/kvm-device-plugin/03-daemonset.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  annotations:
+    image.openshift.io/triggers: |
+      [
+        {"from":{"kind":"ImageStreamTag","name":"kvm-device-plugin:bin"},"fieldPath":"spec.template.spec.containers[?(@.name==\"kvm-device-plugin\")].image"}
+      ]
+  labels:
+    app: kvm-device-plugin
+  name: kvm-device-plugin
+  namespace: kvm-device-plugin
+spec:
+  selector:
+    matchLabels:
+      name: kvm-device-plugin
+  template:
+    metadata:
+      labels:
+        name: kvm-device-plugin
+    spec:
+      containers:
+      - name: kvm-device-plugin
+        image: kvm-device-plugin:latest
```

Comment:
Isn't this in namespace/name format, i.E. `kvm-device-plugin/kvm-device-plugin:latest`?